### ISSUE TITLE
Unsupported KEX algorithm

### ISSUE THREAD

**Issue by marcysutton** [dolmen/github-keygen#25]
In trying to ssh to an enterprise Git server after using your utility, I get an `Unsupported KEX algorithm` error. 
```
Unsupported KEX algorithm "curve25519-sha256@libssh.org"
/Users/marcy/.ssh/config line 33: Bad SSH2 KexAlgorithms 'curve25519-sha256@libssh.org,diffie-hellman-group-exchange-sha256,diffie-hellman-group-exchange-sha1,diffie-hellman-group14-sha1'.
```
Perhaps this is caused by the remote server having a different configuration than Github's? I don't have much more in the way of details, so I'm not sure how helpful this bug report is...

**Comment by dolmen** [comment]
This is because your SSH build (or the library it uses for algorithms) does not have that algorithm enabled. So, the option value set by github-keygen is not recognized by OpenSSH when it parses your ~/.ssh/config.

A few months ago I've implemented a filter to only use algorithms and options that are supported by the installed OpenSSH. Unfortunately this is still only available in the in the `develop` branch. I'll make a release in the following days.

**Comment by dolmen** [comment]
I just released v1.100. Check it out and run github-keygen, this should fix your issue. Keep a copy of the github-keygen output if this is not the case.

**Comment by dolmen** [comment]
@marcysutton Could you check that your issue is fixed?
